### DISCUSSION TITLE
Hostinger domain not getting assigned to vercel project

### DISCUSSION THREAD

**Discussion by ayushs1214** [vercel/community#3574] [Help]
### Summary

I have my project deployed and built on vercel. but when I am trying to map my domain which I bought from HOSTINGER it isn't getting added. I added all the TXT records correctly but still it isn't getting mapped. My txt records have been propogated but I am not getting the option of A record that vercel generally gives after the TXT records are added. And hence my domain is not getting activated. 
What to do about this, I am in an urgent need to deploy it.

### Steps to Reproduce

1. Added my github repo and built it on vercel
2. Trying to map my domain but it isnt getting added
3. Have ben stuck in this from over 4 days now.

**Comment by smaeda-ks** [comment]
@ayushs1214 What the domain name in question? I can take a look
